src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hosting/ClaspLoggingBridge.cs

```diff
@@ -71,4 +71,10 @@ public void LogWarning(string message, params object[] @params)
     {
         return new LspLoggingScope(context, _logger);
     }
+
+    public IDisposable? CreateLanguageContext(string? language)
```

Comment:
I'm guessing this needs the Roslyn version bump.

Reply:
Yes, this is an API break that is coming with the bump